cmd/polisatomicswap/main.go

```diff
@@ -0,0 +1,1142 @@
+// Copyright (c) 2017 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"crypto/rand"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/polissuite/gopolis/chaincfg"
+	"github.com/polissuite/gopolis/chaincfg/chainhash"
+	rpc "github.com/polissuite/gopolis/rpcclient"
+	"github.com/polissuite/gopolis/txscript"
+	"github.com/polissuite/gopolis/wire"
+	"github.com/polissuite/gopolisutil"
+	"github.com/polissuite/gopoliswallet/wallet/txrules"
+	"golang.org/x/crypto/ripemd160"
+)
+
+const verify = true
+
+const secretSize = 32
+
+const txVersion = 2
```

Comment:
Current txVersion is 2. Several modifications are being worked on the develop branch of the daemon. On master you will see v1, but on develop you will see v2. The one used for testing is develop branch.

Reply:
What does version 2 change?

Reply:
Upstream integration of BIP68, BIP112 and BIP113.

Reply:
OK -- I apologize for my harsh comments earlier because I expected this was being developed for the latest release, and this was the problem I first found that really irked me.